README.md

```diff
@@ -46,8 +46,73 @@ be assigned. |
 
 `user_visits_form_smoke_spec.rb` will run an end to end acceptance test, creating a real ticket in Zendesk when this test submits the config form.
 
-The test is run within Docker. This allows us to create an environment that includes the dependencies required by Selenium to run on our Jenkins setup. The Jenkins job that runs the smoke test is triggered every 5 minutes and after this application is deployed.
+The test is run within Docker. This allows us to create an environment that includes the dependencies required by Selenium to run on our Jenkins setup. The [Jenkins job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-smoke-test/) that runs the smoke test is triggered every 5 minutes and after this application is deployed.
 
 The smoke test has been filtered out of the main test run (`bundle exec rspec spec`) via a `smoke` filter.
 
 To run the smoke test, use: `./smoke-test.sh`.
+
+
+## Deploying
+
+The app is deployed to the PaaS and is called `verify-environment-access`. The production space is called `docs`. There are two other spaces, `onboarding-forms` and `sandbox`, but the app is not currently deployed to those.
+
+The environment variables above currently need to be set manually before deployment:
+
+```
+cf login
+```
+
+Choose the `docs` space from the options provided, or target it manually:
+
+```
+cf target -s docs
+```
+
+To set an environment variable:
+
+```
+cf set-env verify-environment-access <ENV_VAR> <VALUE>
+```
+
+To see the current environment config for the app:
+
+```
+cf env verify-environment-access
+```
+
+Once a PR is merged to master this triggers a Jenkins [build job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-build/) and then a [deploy job](https://build.ida.digital.cabinet-office.gov.uk/job/verify-rp-environment-config-form-deploy/).
```

Comment:
Add a paragraph on how to restart the app if we are just changing variables and no PR is being merged.